### PR TITLE
Fix/docker

### PR DESCRIPTION
**Pull request by noameron**
Fixed docker run issues

### CODE REVIEW

**Comment by noameron** [comment]
Hey @nicobrenner I fixed an issue I saw when trying to run the project with docker-compose

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -5,13 +5,14 @@
 version: "3.7"
 services:
   app:
+    # Set container & image name
+    container_name: commandjobs
+    image: commandjobs:1.0
```
Comment (nicobrenner) [inline docker/docker-compose.yml:10]: 👍🏼

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -29,3 +29,4 @@ services:
     stdin_open: true  # Keep STDIN open
 
     working_dir: /commandjobs
+    entrypoint: ["sh", "/commandjobs/docker/docker-entrypoint.sh"]
```
Comment (nicobrenner) [inline docker/docker-compose.yml:32]: 😄

File: docker/docker-entrypoint.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 echo "Starting the application..."
 pip3 install -r config/requirements.txt || echo "Error, could not install requirements.txt $?"
-python3 src/menu.py || echo "Python script exited with error code $?"
+exec python3 src/menu.py || echo "Python script exited with error code $?"
```
Comment (nicobrenner) [inline docker/docker-entrypoint.sh:4]: 🙌🏽

**Review by nicobrenner — APPROVED** [review]
Excellent fix and cleanup. Thank you 🙏🏽

**Comment by noameron** [comment]
Thanks! im still seeing instability in the Workday scraper

**Comment by nicobrenner** [comment]
> Thanks! im still seeing instability in the Workday scraper

What are the symptoms of the instability? Do you see it in capturing the data, in processing and storing the results, or in finding good matches for the resume?

**Pull request merged**